packages/server/tests/postman.json

```
@@ -0,0 +1,820 @@
+		  "host": [
+			"{{base_url}}"
+		  ]
+		}
+	  },
+	  "response": []
+	},
+	{
+	  "name": "net_listening",
+	  "event": [
+		{
+		  "listen": "test",
+		  "script": {
+			"exec": [
+			  "pm.test(\"Success\", () => {",
+			  "    var response = pm.response.json();",
+			  "    pm.expect(response.result).to.equal(\"false\");",
+			  "    pm.expect(response.id).to.equal(\"test_id\");",
+			  "    pm.expect(response.jsonrpc).to.equal(\"2.0\");",
+			  "});",
+			  ""
+			],
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"net_listening\",\n    \"params\": []\n}",
+		  "options": {
+			"raw": {
+			  "language": "json"
+			}
+		  }
+		},
+		"url": {
+		  "raw": "{{base_url}}",
+		  "host": [
+			"{{base_url}}"
+		  ]
+		}
+	  },
+	  "response": []
+	},
+	{
+	  "name": "net_version",
+	  "event": [
+		{
+		  "listen": "test",
+		  "script": {
+			"exec": [
+			  "pm.test(\"Success\", () => {",
+			  "    var response = pm.response.json();",
+			  "    pm.expect(response.result).to.match(/^0x[a-f0-9]+$/);",
+			  "    pm.expect(response.id).to.equal(\"test_id\");",
+			  "    pm.expect(response.jsonrpc).to.equal(\"2.0\");",
+			  "});",
+			  ""
+			],
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"net_version\",\n    \"params\": []\n}",
+		  "options": {
+			"raw": {
+			  "language": "json"
+			}
+		  }
+		},
+		"url": {
+		  "raw": "{{base_url}}",
+		  "host": [
+			"{{base_url}}"
+		  ]
+		}
+	  },
+	  "response": []
+	},
+	{
+	  "name": "web3_clientVersion",
+	  "event": [
+		{
+		  "listen": "test",
+		  "script": {
+			"exec": [
+			  "pm.test(\"Success\", () => {",
+			  "    var response = pm.response.json();",
+			  "    pm.expect(response.result).to.match(/^relay[/]/);",
+			  "    pm.expect(response.id).to.equal(\"test_id\");",
+			  "    pm.expect(response.jsonrpc).to.equal(\"2.0\");",
+			  "});",
+			  ""
+			],
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"web3_clientVersion\",\n    \"params\": []\n}",
+		  "options": {
+			"raw": {
+			  "language": "json"
+			}
+		  }
+		},
+		"url": {
+		  "raw": "{{base_url}}",
+		  "host": [
+			"{{base_url}}"
+		  ]
+		}
+	  },
+	  "response": []
+	}
+  ],
+  "event": [
+	{
+	  "listen": "prerequest",
+	  "script": {
+		"type": "text/javascript",
+		"exec": [
+		  ""
+		]
+	  }
+	},
+	{
+	  "listen": "test",
+	  "script": {
+		"type": "text/javascript",
+		"exec": [
+		  ""
+		]
+	  }
+	}
+  ],
+  "variable": [
+	{
+	  "key": "base_url",
+	  "value": "34.122.35.80:7546",
```

```suggestion
	  "value": "locahost:7546",
```
or
```suggestion
	  "value": "127.0.0.1:7546",
```
Either but should be consistent with what you put in README